src/test/java/org/json/junit/JSONObjectTest.java

```diff
@@ -3476,19 +3476,6 @@ public void issue654IncorrectNestingNoKey2() {
         fail("Excepected Exception.");
     }
     
-    /**
-    * Tests for stack overflow. See https://github.com/stleary/JSON-java/issues/654
-    */
-    @Test(expected = JSONException.class)
-    public void issue654StackOverflowInputWellFormed() {
-        //String input = new String(java.util.Base64.getDecoder().decode(base64Bytes));
-        final InputStream resourceAsStream = JSONObjectTest.class.getClassLoader().getResourceAsStream("Issue654WellFormedObject.json");
-        JSONTokener tokener = new JSONTokener(resourceAsStream);
-        JSONObject json_input = new JSONObject(tokener);
-        assertNotNull(json_input);
-        fail("Excepected Exception.");
```

Comment:
Test should not be removed

Reply:
Why?

Reply:
It is not good remove tests after code changes